examples/advanced/multi-brand-multi-platform/build.js

```diff
@@ -64,4 +64,4 @@ console.log('Build started...');
 })
 
 console.log('\n==============================================');
-console.log('\nBuild completed!');
\ No newline at end of file
+console.log('\nBuild completed!');
```

Comment:
Hmm seems like GitHub did this, not sure if intended?

Reply:
Might be some sort of automatic code formatting thing, not a huge deal.